docs/maya/activation.md

```diff
@@ -0,0 +1,78 @@
+# AdnActivation
+
+The AdnActivation node is a dependency node that allows operations on activation values for the computation of a final activation value that can be used to drive, for example, muscle activations. An example of its use case could be the merging of various sensor activations into one single driven activation to activate a muscle given multiple poses.
```

Comment:
The description is perfect, honestly, but it sounded repetitive because of the "activation" occurrences. Just an idea:

```suggestion
The AdnActivation node is a dependency node that allows operations on a set of input values for the computation of a final value that can be used to drive, for example, muscle activations. An example of its use case could be the merging of the output value of various sensor together to activate a muscle given multiple poses.
```

Reply:
Good for me too. Thanks!